File: MY_CODES/javacode/Positive.java
import java.util.Scanner;

public class Positive {
    public static void main(String[]args)
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the number:");
        int n=sc.nextInt();
        if(n>0)
        {
            System.out.println("Postive number:");
        }
        else if(n<0)
        {
            System.out.println("Negative number:");
        }
        else
        {
            System.out.println("Zero number");
        }

        
    }
}

File: MY_CODES/c programing/Positive.c
#include<stdio.h>
int main()
{
	int i;
	printf("Enter the number:");
	scanf("%d",&i);
	if(i>=0)
	{
		printf("Positive number");
	}
	else
	{
		printf("Negative number:");
	}
	return 0;
}